lib/vagrant-service-manager/command.rb

```diff
@@ -8,14 +9,15 @@ module ServiceManager
     SCCLI_SERVICES = ['openshift', 'k8s']
     KUBE_NAMES = ['kubernetes', 'k8s']
     KUBE_SERVICES = [
-      "etcd",
-      "kube-apiserver",
-      "kube-controller-manager",
-      "kube-scheduler",
-      "kubelet",
-      "kube-proxy",
-      "docker"
+      'etcd', 'kube-apiserver', 'kube-controller-manager', 'kube-scheduler',
+      'kubelet', 'kube-proxy', 'docker'
     ]
```

Comment:
I kind of like this format of listing services. Some coding standard prefer this as this is more readable.  Is there any reason we moved away from it?